### DISCUSSION TITLE
Confusion in 'SMART_BANNER' ad size

### DISCUSSION THREAD

**Discussion by vigamage** [admob-plus/admob-plus#334] [Q&A]
I just created a banner ad in ionic 5 project as below.

```
const banner = new this.admob.BannerAd({
       adUnitId: environment.admob.addUnits.appComponentBottomBannerId,
       size: AdSizeType.SMART_BANNER
});
await banner.show();
```
As google describes [here](https://developers.google.com/admob/android/banner),  the size of the SMART_BANNER should be Screen width x 32|50|90.

but in my case, in my device, I get a 320*100 ad . (test ads)

Anything wrong here? It looks ugly with two black colored margins that covers up device width.

**Comment by ratson** [comment]
@vigamage I am not sure about that.
Would you try if you get the same size with the example https://github.com/googleads/googleads-mobile-ios-examples/tree/master/Swift/admob/BannerExample?

**Reply by vigamage** [reply]
any android one to try? I dont have resources to make an ios app.

**Reply by ratson** [reply]
@vigamage Here is the Android example, https://github.com/googleads/googleads-mobile-android-examples/tree/master/java/admob/BannerExample
